composer.json

```diff
@@ -10,5 +10,14 @@
     "issues": "https://www.drupal.org/project/issues/webform_civicrm",
     "source": "http://cgit.drupalcode.org/webform_civicrm"
   },
-  "require": { }
+  "require": {
+      "drupal/webform": "^5.0"
+  },
+  "extra": {
+    "patches": {
+        "civicrm/civicrm-core": {
+            "Fix Drupal 8+ settings location during tests": "https://patch-diff.githubusercontent.com/raw/civicrm/civicrm-core/pull/18843.patch"
```

Comment:
We will need this until the next minor release, I think. I'm not familiar with the CiviCRM release schedule.